index.html

```diff
@@ -20,6 +20,16 @@
 
     </head>
     <body >
+	    
+	    
+	    <div class="lang_switcher">
+        <center>
+            <button type="submit" name="lang" value="Irish" onclick="Irish()">Gaeilge</button>
+            <button type="submit" name="lang" value="English" onclick="English()">English</button>
+        </center>
+    </div>
```

Comment:
Adding some effects for buttons makes the webpage look good.

Reply:
Adding some effects for buttons makes the webpage look good.